pkg/bundle/bundle_test.go

```diff
@@ -1273,18 +1273,26 @@ func Test_Reconcile(t *testing.T) {
 		},
 	}
 
+	deepCopyArray := func(arr []client.Object) []client.Object {
```

Comment:
I had to add this function to fix a race condition caused by https://github.com/kubernetes-sigs/controller-runtime/pull/1306.